packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1067,6 +1067,35 @@ oie.registration.form.submit = Sign Up
 oie.user.profile.lastname= Last name
 oie.user.profile.firstname= First name
 oie.user.profile.primary.email= Email
+oie.user.profile.login = Username
+oie.user.profile.middleName = Middle name
+oie.user.profile.honorificPrefix = Honorific prefix
+oie.user.profile.honorificSuffix = Honorific suffix
+oie.user.profile.primaryEmail = Primary email
+oie.user.profile.title = Title
+oie.user.profile.displayName = Display name
+oie.user.profile.nickName = Nickname
+oie.user.profile.profileUrl = Profile Url
+oie.user.profile.secondEmail = Secondary email
+oie.user.profile.mobilePhone = Mobile phone
+oie.user.profile.primaryPhone = Primary phone
+oie.user.profile.streetAddress = Street address
+oie.user.profile.city = City
+oie.user.profile.state = State
+oie.user.profile.zipCode = Zip code
```

Comment:
If this is intended for a non-US audience (and really, we should be working towards making UD have globalized usage patterns), then this should be "Postal code"

If we are only targeting US audiences, this should be "ZIP Code" (ZIP is an acronym for Zone Improvement Plan, so capitalized, and "ZIP Code" is a service mark, so needs both items capitalized) .